Shared/src/com/pacoapp/paco/shared/model2/SQLQuery.java

```diff
@@ -108,6 +108,11 @@ public Builder fullEventAndOutputs(Boolean fullEventAndOutputs) {
     }
 
     private Builder addDefaultValues(SQLQuery obj) {
+      // default projection is *
+      if (obj.getProjection() == null) {
+        obj.projection = new String[] { Constants.STAR };
+        obj.fullEventAndOutputs = true;
+      } 
       // find if there is a distinct clause
       boolean isDistinct = false;
       for(String s : obj.getProjection()) {
```

Comment:
The for block will throw a NullPointerException i the obj.getProtection() returns null.